exampleSite/config.toml

```diff
@@ -29,7 +29,7 @@ googleAnalytics = ""
     favicon_png = "favicon.png"
     favicon_svg = "favicon.svg"
     css = []
-    js  = []
+    js  = ["main.js"]
```

Comment:
This is just for the demo. I should rename the entry and output file though. User needs to add their js file in general so the code loaded. The code should be built since it most likely contains JSX.